src/chromeDebugAdapter.ts

```diff
@@ -347,6 +332,37 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
         this._chromeProc = null;
     }
 
+    private async killChromeOnWindows(chromePID: number): Promise<void> {
+        let taskkillCmd = `taskkill /PID ${chromePID}`;
+        logger.log(`Killing Chrome process by pid: ${taskkillCmd}`);
+        try {
+            execSync(taskkillCmd);
+        } catch (e) {
+            // The command will fail if process was not found. This can be safely ignored.
+        }
+
+        for (let i = 0 ; i < 10; i++) {
```

Comment:
What's the use for this for loop? How is this better than using a timeout?

Reply:
This follows the pattern in `findNewlyLaunchedChromeProcess()` at the bottom of the file. It will try up to 10 times and wait 200ms between tries. Total time will be 2000ms in addition to command execution time. It does use a timeout to wait.

I think it's readable but I am open to changing it.

Reply:
This is fine but you could also use the `retryAsync` helper in core's utils.ts if you want.